scripts/plot_perts_x.jl

```diff
@@ -0,0 +1,194 @@
+using Revise
+using Bolt
+using Plots
+using Printf
+using Interpolations
+using DelimitedFiles
+
+# bg/ion setup
+𝕡 = CosmoParams()
+n_q=15
+logqmin,logqmax = -6,-1
+bg = Background(𝕡; x_grid=-20.0:0.004:0.0, nq=n_q)
+𝕣 = Bolt.RECFAST(bg=bg, Yp=𝕡.Y_p, OmegaB=𝕡.Ω_b)
+ih = IonizationHistory(𝕣, 𝕡, bg)
+x_grid = collect(-20:0.004:0.0)
+
+#choose k mode from the 3 I have been looking at
+kp03 =  1000bg.H₀*.3/.333 /10
+kp3 = kp03*10
+k1 = kp3 / .3
+k = kp03 #k1#kp3  #comment/uncomment this line to swap mode comparisons
+#fudge factors to match the CLASS k mode values for the saved comparisons
+fudge_class_03 = 0.029158189805725376/0.030030030030030026 #fudge factor so we get the right .03
```

Comment:
I don't fully understand this -- what's this about again? These are some weird numbers.

Reply:
I was sloppy in doing the class comparison and didn't make sure the k-modes matched properly ahead of time (and so fudged it afterward). The k-modes used for the comparison by Bolt are now loaded from the CLASS file (they were already in there) to avoid this silliness.